subiquity/common/filesystem/manipulator.py

```diff
@@ -197,11 +197,12 @@ def partition_disk_handler(self, disk, spec, *, partition=None, gap=None):
 
             # adjust downward the partition size (if necessary) to accommodate
             # bios/grub partition
-            if spec['size'] > gaps.largest_gap_size(disk):
+            gap = gaps.largest_gap(disk)
```

Comment:
I don't know if this is overthinking, but is it worth adding a comment that in this case it's safe to ignore the passed-in gap because there can only be one?

Reply:
I like this, I had to stare at the code for a bit to convince myself that assigning a new gap was ok.  Will do.